pyreisejl/utility/call.py

```diff
@@ -21,15 +23,66 @@ def _record_scenario(scenario_id, runtime):
     # Update status in ExecuteList.csv on server
     insert_in_file(const.EXECUTE_LIST, scenario_id, "status", "finished")
 
-    hours, minutes, seconds = sec2hms(runtime)
+    hours, minutes, _ = sec2hms(runtime)
     insert_in_file(
         const.SCENARIO_LIST, scenario_id, "runtime", "%d:%02d" % (hours, minutes)
     )
 
 
+cols = {
+    "branch": ["branch_id", "from_bus_id", "to_bus_id", "x", "rateA"],
+    "dcline": ["dcline_id", "from_bus_id", "to_bus_id", "Pmin", "Pmax"],
+    "bus": ["bus_id", "Pd", "zone_id"],
+    "plant": [
+        "plant_id",
+        "bus_id",
+        "status",
+        "Pmin",
+        "Pmax",
+        "type",
+        "ramp_30",
+        "GenFuelCost",
+        "GenIOB",
+        "GenIOC",
+        "GenIOD",
+    ],
+}
+
+drop_cols = {"gencost": ["plant_id", "interconnect"]}
```

Comment:
Why do we drop columns for `gencost` instead of selecting columns in `cols` as we do for the other data frames?

Reply:
I guess it's shorter, i.e. between a white list in selecting columns vs. a black list in dropping columns, always go for the shorter one?

Reply:
I see. Then you have the issue of considering columns that are not needed if we augment the `gencost` data frame in the future for some reason.

Reply:
> I see. Then you have the issue of considering columns that are not needed if we augment the `gencost` data frame in the future for some reason.

Indeed. But it is unlikely the case unless we would like to support different cost curve types for different entries.

Reply:
Yeah, only reason is that was the first thing I thought of and it's probably shorter to drop cols from gencost compared with the other ones. It also kinda made sense when just looking at types.jl where we assign the whole gencost matrix directly, compared to the other fields which are assigned by column name.